events/wc_dark_cult_events.txt

```diff
@@ -889,16 +888,33 @@ narrative_event = {
 	border = GFX_event_narrative_frame_intrigue
 
 	is_triggered_only = yes
-	
+
 	immediate = {
 		add_trait = travelling
 	}
 
 	# I will fulfill the will of the Gods!
 	option = {
 		name = EVTOPTA_WCDAC_5030
-
-		hidden_effect = { 73 = { province_event = { id = WCDAC.5034 days = 60 } } }
+		trigger_switch = {
+				on_trigger = has_character_modifier
+				follower_of_yogg_saron = { remove_character_modifier = follower_of_yogg_saron }
+				follower_of_nzoth = { remove_character_modifier = follower_of_nzoth }
+		}
+		add_character_modifier = { name = follower_of_cthun duration = -1 }
+		set_global_flag = twilights_hammer_kalimdor
+	}
+	option = {
+		name = EVTOPTB_WCDAC_5030
+		trigger_switch = {
+				on_trigger = has_character_modifier
+				follower_of_cthun = { remove_character_modifier = follower_of_cthun }
+				follower_of_nzoth = { remove_character_modifier = follower_of_nzoth }
+		}
+		add_character_modifier = { name = follower_of_yogg_saron duration = -1 }
+	}
+	after = {
+		hidden_effect = { 73 = { province_event = { id = WCDAC.5034 years= 5 } } }
```

Comment:
It's too long to wait, I would change it to old values.

Reply:
it was actually too short